newIDE/app/src/EventsSheet/EventsTree/index.js

```diff
@@ -665,6 +749,7 @@ export default class ThemableEventsTree extends Component<EventsTreeProps, *> {
             ref: list => (this._list = list),
             onScroll: this.props.onScroll,
           }}
+          slideRegionSize={0}
```

Comment:
I think this also reduce the region size on desktop, let's keep it to a good value and investigate what we can do for mobile/touchscreens :) Maybe there is a way to avoid being "stuck".

Reply:
Yes, I saw that after committing 😅 
I tried the method of drag&scroll using drop targets (see WIP commit) but it looks like it may not be a problem of rst like I thought as the problem persists. (at least it can scroll up)